README.md

```diff
@@ -105,41 +120,60 @@ buildscript {
 }
 ```
 
-Don't forget to apply the plugin:
+#### Applying the plugin
+
+Next, apply the plugin to your project.
 
+You can setup serialization plugin with the kotlin plugin using [Gradle plugins DSL](https://docs.gradle.org/current/userguide/plugins.html#sec:plugins_block) **instead** of traditional `apply plugin`:
```

Comment:
If one using `plugins {}` block, there's no need to add plugins as build script dependencies – plugins are resolved directly from Gradle Plugin Portal via plugin markers. Therefore I'd recommend moving this section before `Adding the plugin to classpath` and probably make it on a higher level (`###`) or join sections 'adding the plugin to the classpath' and 'applying the plugin in an old way' as it was before.

Reply:
I have just tried it out and if I remove the
`classpath(kotlin("serialization", version = kotlinVersion))` from my buildscript dependencies then I get the following error:
```
Plugin [id: 'kotlinx-serialization'] was not found in any of the following sources:

- Gradle Core Plugins (not a core plugin, please see https://docs.gradle.org/6.1-milestone-2/userguide/standard_plugins.html for available core plugins)
- Plugin Repositories (plugin dependency must include a version number for this source)
```